LABS-EU3/quality_hub_frontend
# Hotfix

##  What does this PR do
This pull request is for coach and appointment cards styling

##  Description of task to be completed
Improve coach and appointment card spacing 
Remove duplicate loader from appointments
Center loaders on pages

##  How should this manually be tested
Go to dashboard and marketplace, check out cards. Check loaders are centered on pages

##  What are the relevant Trello board stories
improve appointment card spacing/layout

##  Screenshots (If appropriate)


##  Questions (If any):
